e2e/nomostest/git-server.go

```diff
@@ -63,7 +63,15 @@ func setupGit(nt *NT) error {
 			return err
 		}
 		nt.caCertPath = caCertPath
+	}
+	if *e2e.RegistryProvider == e2e.Local {
+		if err := installRegistryServer(nt); err != nil {
```

Comment:
any way we can do this in paralle with the git server, since they're both waiting for reconcile?

Reply:
Yeah they can be done in parallel. Updated